src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.cs

```diff
@@ -59,6 +58,322 @@ public object ScrollOwner
 
 		private ScrollViewer Scroller => ScrollOwner as ScrollViewer;
 
+		internal object ManagedContent
+		{
+			get
+			{
+#if __ANDROID__ || __IOS__
+				if (Content is NativeScrollContentPresenter nativePresenter)
+				{
+					return nativePresenter.Content;
+				}
+#endif
+
+				return Content;
+			}
+		}
+
+		protected override void OnBringIntoViewRequested(BringIntoViewRequestedEventArgs args)
+		{
+			base.OnBringIntoViewRequested(args);
+
+			UIElement content = ManagedContent as UIElement;
+
+			if (args.Handled ||
+				args.TargetElement == this ||
+				(args.TargetElement == content && content?.Visibility == Visibility.Collapsed) ||
+				!SharedHelpers.IsAncestor(args.TargetElement, this, true /*checkVisibility*/))
+			{
+				// Ignore the request when:
+				// - There is no InteractionTracker to fulfill it.
```

Comment:
I don't see this in the conditions above, is this relevant for Uno? Is this a TODO?

Reply:
It is just a copy of the comment from MUX repo, not relevant to us.

Reply:
Could we then add a small comment to explain that it's on purpose for Uno :)